stats/stats.py

```diff
@@ -43,3 +43,11 @@ def get_most_changed_files(self, number_of_files=10):
         for _, stat in self.repo_stats:
             stats[stat.filename] += 1
         return stats.most_common(number_of_files)
+
+    def bars_and_spaces(self, item_dict, change_count):
+        max_change_count = max(v for k,v in item_dict)
+        bar_length = int(36*(change_count/max_change_count))
```

Comment:
I would make `36` a constant and/or an argument to the function

Reply:
Added `space_length = 36` as arg.